### PR TITLE
DR UI textual update for the epic 4.15

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
https://bugzilla.redhat.com/show_bug.cgi?id=2255332

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@SanjalKatiyar , @GowthamShanmugam Please review

**Comment by GowthamShanmugam** [comment]
Please share the screenshots

**Comment by TimothyAsirJeyasing** [comment]
image

**Comment by TimothyAsirJeyasing** [comment]
image
image
image
image
image

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx [hunk 1]
```diff
@@ -119,14 +115,10 @@ export const OSDMigrationProgress: React.FC<OSDMigrationProgressProps> = ({
...
               <FlexItem align={{ default: 'alignRight' }}>
-                {t(
-                  '{{ percentageComplete }}% completed ({{ migratedDevices }}/{{ totalOsd }} remaining)',
-                  {
-                    percentageComplete,
-                    migratedDevices,
-                    totalOsd,
-                  }
-                )}
+                {t('{{ migratedDevices }}/{{ totalOsd }} remaining', {
```
Comment (GowthamShanmugam) [inline packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:118]:
```suggestion
                {t('{{ migratedDevices }}/{{ totalOSD }} remaining', {
```

File: packages/ocs/modals/osd-migration/osd-migration-modal.tsx [hunk 2]
```diff
@@ -47,14 +49,14 @@ export const OSDMigrationModal: React.FC<OSDMigrationModalProps> = ({
...
+      title={t('Prepare the cluster for Regional DR setup')}
       isOpen={isOpen}
       onClose={closeModal}
     >
       {t(
-        'Configure the cluster for a Regional-DR setup by migrating OSDs. Migration may take some time depending on several factors. To learn more about OSDs migration best practices and its consequences refer to the documentation.'
+        'To prepare the cluster for Regional DR setup, you must migrate the OSDs. Migrating OSDs may take some time to complete based on your cluster. To learn more about the consequences and best practices for migrating the OSDs, see the link. '
       )}
-      {/* TODO: Show doc link once ViewDocumentation moved to shared */}
+      <ViewDocumentation text={' '} padding={null} doclink={ODF_DR_DOC_HOME} />
```
Comment (GowthamShanmugam) [inline packages/ocs/modals/osd-migration/osd-migration-modal.tsx:59]: text and padding is optional parameters, no need to pass empty

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 1 above)
Comment (Chandan141996) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:118]: @TimothyAsirJeyasing @GowthamShanmugam please use 'OSDs remaining' instead of 'remaining' as shown in the design mockup

{t('{{ migratedDevices }}/{{ totalOSD }} OSDs remaining', {

File: locales/en/plugin__odf-console.json [hunk 3]
```diff
@@ -43,7 +43,7 @@
...
   "Cluster not pre-configured for Regional-DR": "Cluster not pre-configured for Regional-DR",
-  "The selected cluster(s)[{{clusters}}] is not pre-configured for a Regional-DR setup. Migrate the cluster's OSD to optimise it for Disaster recovery services. To learn more about OSDs migration best practices and its consequences refer to the documentation.": "The selected cluster(s)[{{clusters}}] is not pre-configured for a Regional-DR setup. Migrate the cluster's OSD to optimise it for Disaster recovery services. To learn more about OSDs migration best practices and its consequences refer to the documentation.",
+  "The selected cluster(s)[{{clusters}}] is not pre-configured for a Regional-DR setup. Migrate the OSDs to optimise the cluster for disaster recovery services. To learn about the best practices and consequences for OSD migration, refer to the documentation.": "The selected cluster(s)[{{clusters}}] is not pre-configured for a Regional-DR setup. Migrate the OSDs to optimise the cluster for disaster recovery services. To learn about the best practices and consequences for OSD migration, refer to the documentation.",
```
Comment (Chandan141996) [inline locales/en/plugin__odf-console.json:46]: @TimothyAsirJeyasing The error message on the DR policy page in the above screenshot appears to be slightly different. Kindly update it.

**Comment by Chandan141996** [comment]
@TimothyAsirJeyasing The check/success icon is not present against the 'Completed' status, do add it.

**Comment by Chandan141996** [comment]
@TimothyAsirJeyasing @GowthamShanmugam 

General observation
1. In most places, there are unnecessary whitespaces between text and links, please get rid of whitespaces. 
2. Padding between the cluster status, the separator line, and the migration status on the 'Status card' is inappropriate; see mockups or any status card on OCP for reference.

File: packages/ocs/modals/osd-migration/osd-migration-modal.tsx (hunk 2 above)
Comment (TimothyAsirJeyasing) [reply packages/ocs/modals/osd-migration/osd-migration-modal.tsx:59]: image

I have used the text and padding to avoid "View Documentation" (if i do not pass text) and extra space above the link (if i omit passing padding null).

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx [hunk 4]
```diff
@@ -45,7 +45,7 @@ export const OSDMigrationProgress: React.FC<OSDMigrationProgressProps> = ({
   dataLoadError,
 }) => {
   const { t } = useCustomTranslation();
-  const [migratedDevices, totalOsd, percentageComplete] = calculateOSDMigration(
```
Comment (SanjalKatiyar) [inline packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:48]: we don't need to show percentage anymore ?? @GowthamShanmugam @Chandan141996

**Comment by SanjalKatiyar** [comment]
add BZ link to PR description...

File: locales/en/plugin__odf-console.json [hunk 5]
```diff
@@ -43,7 +43,7 @@
   "{{ names }} has either an unsupported ODF version or the ODF operator is missing, install or update to ODF {{ version }} or the latest version to enable DR protection.": "{{ names }} has either an unsupported ODF version or the ODF operator is missing, install or update to ODF {{ version }} or the latest version to enable DR protection.",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:43]:
```suggestion
  "{{ names }} has either an unsupported Data Foundation version or the Data Foundation operator is missing, install or update to Data Foundation {{ version }} or the latest version to enable DR protection.": "{{ names }} has either an unsupported ODF version or the ODF operator is missing, install or update to ODF {{ version }} or the latest version to enable DR protection.",
```

**Comment by SanjalKatiyar** [comment]
do not use term "ODF" or "Openshift" anywhere... use "Data Foundation" instead...

**Comment by GowthamShanmugam** [comment]
> @TimothyAsirJeyasing @GowthamShanmugam
> 
> General observation
> 
> 1. In most places, there are unnecessary whitespaces between text and links, please get rid of whitespaces.
> 2. Padding/Alignment between the cluster status, the separator line, and the migration status on the 'Status card' is inappropriate; see mockups or any status card on OCP for reference.

Please point out the places.

File: packages/ocs/modals/osd-migration/osd-migration-modal.tsx (hunk 2 above)
Comment (GowthamShanmugam) [reply packages/ocs/modals/osd-migration/osd-migration-modal.tsx:59]: i don't think passing empty text is resolving this space issue, it is padding, pass only empty string to padding.

File: locales/en/plugin__odf-console.json [hunk 6]
```diff
@@ -485,10 +485,10 @@
...
-  "Configure the cluster for a Regional-DR setup by migrating OSDs. Migration may take some time depending on several factors. To learn more about OSDs migration best practices and its consequences refer to the documentation.": "Configure the cluster for a Regional-DR setup by migrating OSDs. Migration may take some time depending on several factors. To learn more about OSDs migration best practices and its consequences refer to the documentation.",
-  "Optimise": "Optimise",
+  "(Prepare cluster for DR setup)": "(Prepare cluster for DR setup)",
+  "Prepare the cluster for Regional DR setup": "Prepare the cluster for Regional DR setup",
+  "To prepare the cluster for Regional DR setup, you must migrate the OSDs. Migrating OSDs may take some time to complete based on your cluster. To learn more about the consequences and best practices for migrating the OSDs, see the link. ": "To prepare the cluster for Regional DR setup, you must migrate the OSDs. Migrating OSDs may take some time to complete based on your cluster. To learn more about the consequences and best practices for migrating the OSDs, see the link. ",
```
Comment (GowthamShanmugam) [inline locales/en/plugin__odf-console.json:490]: Is the Document team going to add any section for this? Right now the document link is simply redirecting to the DR configuration section.

File: packages/mco/components/mco-dashboard/data-policy/cluster-app-card/cluster.tsx [hunk 7]
```diff
@@ -209,15 +209,15 @@ export const ApplicationsSection: React.FC<ApplicationsSectionProps> = ({
...
+        {t(' {{ protectedAppCount }} protected', {
           protectedAppCount,
         })}
       </Text>
       <Text className="text-muted">
-        {t(
-          ' {{ appsWithIssues }} of {{ protectedAppCount }} apps with issues',
-          { appsWithIssues, protectedAppCount }
-        )}
+        {t(' {{ appsWithIssues }} of {{ protectedAppCount }} with issues', {
```
Comment (GowthamShanmugam) [inline packages/mco/components/mco-dashboard/data-policy/cluster-app-card/cluster.tsx:217]:
```suggestion
        {t('{{ appsWithIssues }} of {{ protectedAppCount }} with issues', {
```

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 4 above)
Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:48]: In the recent UX microcopy, the percentage is removed as they mentioned that showing any one details is enough as per the current view.

**Comment by TimothyAsirJeyasing** [comment]
image
image
image

**Comment by TimothyAsirJeyasing** [comment]
image

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 1 above)
Comment (Chandan141996) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx]: @TimothyAsirJeyasing Have you updated this yet? Please share a screenshot, so this can be closed.

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 4 above)
Comment (Chandan141996) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:45]: The Microcopy team suggested not to repeat information, the number of OSDs remaining is enough as per their suggestion. Let's hide it in UI, in case we may need it later if someone opens a BZ for that

File: packages/ocs/modals/osd-migration/osd-migration-modal.tsx [hunk 8]
```diff
@@ -47,14 +49,14 @@ export const OSDMigrationModal: React.FC<OSDMigrationModalProps> = ({
...
+      title={t('Prepare the cluster for Regional DR setup')}
       isOpen={isOpen}
       onClose={closeModal}
     >
       {t(
-        'Configure the cluster for a Regional-DR setup by migrating OSDs. Migration may take some time depending on several factors. To learn more about OSDs migration best practices and its consequences refer to the documentation.'
+        'To prepare the cluster for Regional DR setup, you must migrate the OSDs. Migrating OSDs may take some time to complete based on your cluster. To learn more about the consequences and best practices for migrating the OSDs, see the link. '
       )}
-      {/* TODO: Show doc link once ViewDocumentation moved to shared */}
+      <ViewDocumentation text={' '} padding={''} doclink={ODF_DR_DOC_HOME} />
```
Comment (GowthamShanmugam) [inline packages/ocs/modals/osd-migration/osd-migration-modal.tsx:59]: Migrating OSD is at ODF cluster level, so the consequences has been documented for the ODF cluster level.  But the document which your are linking is DR document.

Comment (GowthamShanmugam) [reply packages/ocs/modals/osd-migration/osd-migration-modal.tsx:59]: Is the document team is not going to document the risk involved in migration?

Comment (TimothyAsirJeyasing) [reply packages/ocs/modals/osd-migration/osd-migration-modal.tsx:59]: Checked with the doc team, Currently, they do not have any documentation/content related to migrating OSDs. So they are suggesting to omit the link from the UI. Will it be ok to have this link now and i can fix it later otherwise we need to remove the link 
@Chandan141996  ^^

Comment (GowthamShanmugam) [reply packages/ocs/modals/osd-migration/osd-migration-modal.tsx:59]: it is better to remove the link

Comment (GowthamShanmugam) [reply packages/ocs/modals/osd-migration/osd-migration-modal.tsx:59]: also, i can see docu link in other error messaged too

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 4 above)
Comment (GowthamShanmugam) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:45]: Then modify the calculateOSDMigration function, it is still calculating percentage.

**Comment by TimothyAsirJeyasing** [comment]
image
Screenshot from 2024-01-16 14-14-19

**Comment by TimothyAsirJeyasing** [comment]
image

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx [hunk 9]
```diff
@@ -28,12 +28,12 @@ const calculateOSDMigration = (
     return [null, null, null];
   }
   const migratedDevices = getCephStoreType(cephData)?.[BLUESTORE_RDR] || 0;
-  const totalOsd =
+  const totalOSD =
     (getCephStoreType(cephData)?.[BLUESTORE] || 0) + migratedDevices;
   const percentageComplete =
```
Comment (GowthamShanmugam) [inline packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:33]: calculateOSDMigration is creating percentageComplete but no where it is used

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx [hunk 10]
```diff
@@ -81,48 +81,34 @@ export const OSDMigrationProgress: React.FC<OSDMigrationProgressProps> = ({
               <HealthItem
                 icon={<InProgressIcon className="co-dashboard-icon" />}
                 state={HealthState.OK}
```
Comment (SanjalKatiyar) [inline packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:83]: why is state `HealthState.OK` when migration is in progress ?? cc @GowthamShanmugam (fyi)

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx [hunk 11]
```diff
@@ -81,48 +81,34 @@ export const OSDMigrationProgress: React.FC<OSDMigrationProgressProps> = ({
...
                   <RedExclamationCircleIcon className="co-dashboard-icon" />
                 }
-                title={t('Could not migrate cluster OSDs.')}
-              />
-            </FlexItem>
-            <FlexItem>
-              <ViewDocumentation
-                text={t('Check documentation')}
-                doclink={ODF_DR_DOC_HOME}
+                state={HealthState.OK}
```
Comment (SanjalKatiyar) [inline packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:103]: why is state `HealthState.OK` when migration failed ?? cc @GowthamShanmugam (fyi)

File: packages/ocs/dashboards/persistent-internal/status-card/status-card.tsx [hunk 12]
```diff
@@ -200,12 +200,12 @@ export const StatusCard: React.FC = () => {
...
-            cephData={cephCluster}
-            dataLoaded={loaded}
-            dataLoadError={loadError}
-          />
         </HealthBody>
+        <OSDMigrationProgress
+          cephData={cephCluster}
+          dataLoaded={loaded}
+          dataLoadError={loadError}
+        />
```
Comment (SanjalKatiyar) [inline packages/ocs/dashboards/persistent-internal/status-card/status-card.tsx:208]: why is this needed ??

**Comment by SanjalKatiyar** [comment]
> image 

how does "Details" card look in this state, on the medium/small screens ??

File: packages/ocs/dashboards/persistent-internal/status-card/status-card.tsx (hunk 12 above)
Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/persistent-internal/status-card/status-card.tsx:208]: This is to show the osd migration progress status in the status card

Comment (SanjalKatiyar) [reply packages/ocs/dashboards/persistent-internal/status-card/status-card.tsx:208]: what I mean is why u need to update it again, when it was already added... why these modifications were needed ??

Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/persistent-internal/status-card/status-card.tsx:208]: This is moved outside of HealthBody as its not related to HealthBody. Also after the removal of the doc link, it gives an alignment issue for the second part of the string to right justify.

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 10 above)
Comment (SanjalKatiyar) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:80]: and will `HealthState.OK` not have message of it's own ?? does `HealthState.OK ` not show any message ?? does it make sense to pass `HealthState.OK` as a prop when state is in progress or failed ??

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 11 above)
Comment (SanjalKatiyar) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:100]: https://github.com/red-hat-storage/odf-console/pull/1157#discussion_r1453380548

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 10 above)
Comment (SanjalKatiyar) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:80]: if it is to avoid sub status then why specifically u need to pass `HealthState.OK` ?? infact do you need to pass anything in the first place ??

Comment (SanjalKatiyar) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:80]: then pass correct status... is there any issue with that ??

Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:80]: As per UX design there is no sub status to be shown right @Chandan141996

File: packages/mco/components/create-dr-policy/select-replication-type.tsx [hunk 13]
```diff
@@ -99,7 +99,7 @@ const getErrorMessage = (
     return {
       message: t('Cluster not pre-configured for Regional-DR'),
       description: t(
-        "The selected cluster(s)[{{clusters}}] is not pre-configured for a Regional-DR setup. Migrate the cluster's OSD to optimise it for Disaster recovery services. To learn more about OSDs migration best practices and its consequences refer to the documentation.",
+        'The selected cluster(s)[{{clusters}}] is not configured for Regional-DR setup. Migrate the OSDs to optimise the cluster for disaster recovery services. To learn about the best practices and consequences for OSD migration, refer to the documentation.',
```
Comment (GowthamShanmugam) [inline packages/mco/components/create-dr-policy/select-replication-type.tsx:102]: this error message is also related to DR migration, And we dont have any document.

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 11 above)
Comment (GowthamShanmugam) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:100]: what is this sub-status thing? setting health is ok is misleading

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx [hunk 14]
```diff
@@ -60,7 +57,7 @@ export const OSDMigrationProgress: React.FC<OSDMigrationProgressProps> = ({
       <Flex alignItems={{ default: 'alignItemsCenter' }}>
         {migrationStatus === OSDMigrationStatus.COMPLETED && (
           <>
-            <FlexItem>
+            <FlexItem className="pf-m-spacer-none">
```
Comment (GowthamShanmugam) [inline packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:60]: may i know why this change?

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 10 above)
Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:80]: Sure will do that, thank you

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 14 above)
Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:60]: its not required now as we removed the doc link

File: packages/mco/components/create-dr-policy/select-replication-type.tsx (hunk 13 above)
Comment (TimothyAsirJeyasing) [reply packages/mco/components/create-dr-policy/select-replication-type.tsx:102]: @Chandan141996 Shall we also remove the word "refer to the documentation" here ?

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 1 above)
Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx]: yes its done, 
image

File: packages/ocs/modals/osd-migration/osd-migration-modal.tsx (hunk 2 above)
Comment (TimothyAsirJeyasing) [reply packages/ocs/modals/osd-migration/osd-migration-modal.tsx]: done

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 4 above)
Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:45]: @GowthamShanmugam  Not getting your point, Could you please give more light on this, I have remove the percentage calculation already as per another comment, We can add it later if required, Will it be ok ?

File: packages/ocs/modals/osd-migration/osd-migration-modal.tsx (hunk 8 above)
Comment (TimothyAsirJeyasing) [reply packages/ocs/modals/osd-migration/osd-migration-modal.tsx]: done

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 9 above)
Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx]: Ok, I kept it to see if I can use it later,  I deleted it now

**Comment by TimothyAsirJeyasing** [comment]
image

**Comment by SanjalKatiyar** [comment]
> image

I specifically mentioned "Details" card https://github.com/red-hat-storage/odf-console/pull/1157#issuecomment-1893479337, not status card...

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx [hunk 15]
```diff
@@ -80,49 +77,37 @@ export const OSDMigrationProgress: React.FC<OSDMigrationProgressProps> = ({
             <FlexItem className="pf-u-mt-xl">
               <HealthItem
                 icon={<InProgressIcon className="co-dashboard-icon" />}
-                state={HealthState.OK}
-                title={t('Cluster OSDs are being migrated')}
+                state={HealthState.PROGRESS}
```
Comment (SanjalKatiyar) [inline packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:80]: this change is diff from what we discussed... plz check https://github.com/red-hat-storage/odf-console/pull/1157#discussion_r1453431703 again...

Comment (SanjalKatiyar) [inline packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:80]:
```suggestion
```
why r u sending this, when u r passing `disableDetails` ??

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx [hunk 16]
```diff
@@ -80,49 +77,37 @@ export const OSDMigrationProgress: React.FC<OSDMigrationProgressProps> = ({
...
+                disableDetails={true}
```
Comment (SanjalKatiyar) [inline packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:81]:
```suggestion
                disableDetails
```

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx [hunk 17]
```diff
@@ -80,49 +77,37 @@ export const OSDMigrationProgress: React.FC<OSDMigrationProgressProps> = ({
...
           <>
-            <FlexItem>
+            <FlexItem className="pf-u-mt-xl">
               <HealthItem
-                state={HealthState.OK}
                 icon={
                   <RedExclamationCircleIcon className="co-dashboard-icon" />
                 }
-                title={t('Could not migrate cluster OSDs.')}
+                state={HealthState.ERROR}
```
Comment (SanjalKatiyar) [inline packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:101]:
```suggestion
```
why r u sending this, when u r passing `disableDetails` ??

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx [hunk 18]
```diff
@@ -80,49 +77,37 @@ export const OSDMigrationProgress: React.FC<OSDMigrationProgressProps> = ({
...
+                disableDetails={true}
```
Comment (SanjalKatiyar) [inline packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:102]:
```suggestion
                disableDetails
```

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 15 above)
Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:80]: I thought it gives a readability to know about the status quickly, Sure will remove this

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 16 above)
Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx]: done

File: packages/ocs/modals/osd-migration/osd-migration-details.tsx [hunk 19]
```diff
@@ -55,7 +59,7 @@ export const OSDMigrationDetails: React.FC<OSDMigrationDetailsProps> = ({
                 })
               }
             >
-              {t('Optimise cluster')}
+              {t('(Prepare cluster for DR setup)')}
```
Comment (bipuladh) [inline packages/ocs/modals/osd-migration/osd-migration-details.tsx:62]: Regional DR or only DR Setup?

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx (hunk 4 above)
Comment (bipuladh) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:45]: @GowthamShanmugam we can keep it and just not use it.

File: packages/ocs/modals/osd-migration/osd-migration-details.tsx (hunk 19 above)
Comment (TimothyAsirJeyasing) [reply packages/ocs/modals/osd-migration/osd-migration-details.tsx:62]: actually this for any DR creation, so we did not mentioned specifically

Comment (Chandan141996) [reply packages/ocs/modals/osd-migration/osd-migration-details.tsx:62]: We explain it to the user once they see the modal that this operation is for regional DR; the text we have used here is only to keep the action name short and also because there isn't much space.

**Comment by TimothyAsirJeyasing** [comment]
@bipuladh , @GowthamShanmugam , I have already removed the percentageComplete in this pr. I can add it any time if required later.  Otherwise if we need to add back into this PR then i can update this PR, Please suggest me

**Comment by bipuladh** [comment]
> @bipuladh , @GowthamShanmugam , I have already removed the "percentageComplete" in this pull request. I can add it at any time if required later. Otherwise, if we need to reintroduce it into this PR, then I can update it. Please advise.

Please keep the function as it is. It might be useful for someone else.

**Comment by TimothyAsirJeyasing** [comment]
Sure, its done now

image

File: locales/en/plugin__odf-console.json [hunk 20]
```diff
@@ -459,14 +459,14 @@
...
   "Raw capacity is the absolute total disk space available to the array subsystem.": "Raw capacity is the absolute total disk space available to the array subsystem.",
   "Cluster ready for Regional-DR setup.": "Cluster ready for Regional-DR setup.",
   "Setting up disaster recovery": "Setting up disaster recovery",
-  "Cluster OSDs are being migrated": "Cluster OSDs are being migrated",
-  "{{ percentageComplete }}% completed ({{ migratedDevices }}/{{ totalOsd }} remaining)": "{{ percentageComplete }}% completed ({{ migratedDevices }}/{{ totalOsd }} remaining)",
-  "Could not migrate cluster OSDs.": "Could not migrate cluster OSDs.",
-  "Check documentation": "Check documentation",
+  "Migrating cluster OSDs": "Migrating cluster OSDs",
+  "{{ migratedDevices }}/{{ totalOSD }} OSDs remaining": "{{ migratedDevices }}/{{ totalOSD }} OSDs remaining",
+  "Could not migrate cluster OSDs, Check logs": "Could not migrate cluster OSDs, Check logs",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:468]: @Chandan141996 @GowthamShanmugam is this what expected here ?? plain text `, Check logs` (with capital `C`), without any external link or anything ??

Comment (Chandan141996) [reply locales/en/plugin__odf-console.json:468]: My suggestion was to replace the existing `Check documentation` link with a `Check logs` link, so there shouldn't be any other changes than what was suggested

Comment (Chandan141996) [reply locales/en/plugin__odf-console.json:468]: Don't use `comma` here in the design it was a `period`.

Comment (TimothyAsirJeyasing) [reply locales/en/plugin__odf-console.json]: Updated

**Review by GowthamShanmugam — APPROVED** [review]

File: locales/en/plugin__odf-console.json (hunk 20 above)
Comment (SanjalKatiyar) [reply locales/en/plugin__odf-console.json]: this is still not the correct fix as UX suggestion here is to have a logs "link" instead of doc link... but not blocking this PR anymore on this, we can discuss and send a follow up later...
